pyramid_swagger/api.py

```diff
@@ -101,29 +103,104 @@ def view_for_api_declaration(request):
     return view_for_api_declaration
 
 
-def resolve_ref(spec, url):
+def dereference_definition(spec, url, current_file, definitions_dict):
+    """
+    Dereference a given definition (by URL) updating the definitions
+    dictionary and fetching all the related objects
+    :param spec:
+    :param url: url of the reference to be extracted
+    :param current_file: base file used for relative url resolution
+    :param definitions_dict: dictionary containing all the definitions
+    :return: python object containing the model representation specified in url
+    """
+
+    def _translate_definition_target(target, prefix='', separator='|'):
+        """
+        Translate the definition target to a consistent way
+        which does not uses the '#' for the name definition
+        :param target: original definition target
+        :param prefix: prefix used if no file were specified on the target
+        :param separator: separator character to be used instead of '#'
+        :return: the converted target (it will be the new name of the link)
+        """
+        value = target.replace("#/definitions/", separator)
+        if value.split(separator)[0] == '':
```

Comment:
Isn't this the same as `if value[0] == separator:` as long as separator is a char?